src/accessors.d

```diff
@@ -84,24 +84,24 @@ template GenerateReader(string name, alias field)
     static enum helper()
     {
         import std.string : format;
-        import std.traits : ForeachType, isArray, isSomeString, MutableOf;
+        import std.traits : isArray, isSomeString;
 
         enum visibility = getVisibility!(field, Read);
-        enum outputType = typeName!(typeof(field));
         enum accessorName = accessor(name);
 
         static if (isArray!(typeof(field)) && !isSomeString!(typeof(field)))
         {
-            enum valueType = typeName!(MutableOf!(ForeachType!(typeof(field))));
-
-            return format("%s final @property inout(%s)[] %s() inout " ~
-                "{ inout(%s)[] result = null; result ~= this.%s; return result; }",
-                visibility, valueType, accessorName, valueType, name);
+            return format("%s final @property auto %s() inout {"
+                        ~ "import std.traits;"
```

Comment:
_Local imports should specify the symbols being imported to avoid hiding local symbols._